test/pact/connector-client/connector-post-create-charge.pact.test.js

```diff
@@ -49,7 +50,7 @@ describe('connector client', function () {
         return provider.addInteraction(
           new PactInteractionBuilder(`${CHARGES_RESOURCE}/${gatewayAccountId}/charges`)
             .withUponReceiving('a valid post create charge request')
-            .withState('a stripe gateway account with external id 42 exists in the database')
+            .withState('a Worldpay gateway account with id 3456, gateway account credentials with external_id creds123 exists')
```

Comment:
On L42 you have Stripe - but in this sentence you say Worldpay.
Can you please check
https://github.com/alphagov/pay-selfservice/pull/3944/files#diff-69d691eb41a998eacfe7eb24fbb00783f76ced1d2f95f6e133225efe883a6f04R44

Reply:
Good spot. Removed payment_provider as it is no longer required